packages/cardpay-cli/reward-pool.ts

```diff
@@ -13,24 +13,16 @@ export async function rewardTokenBalances(
 ): Promise<void> {
   let web3 = await getWeb3(network, mnemonic);
   let rewardPool = await getSDK('RewardPool', web3);
-  if (rewardProgramId) {
-    const tokenBalances = await rewardPool.rewardTokenBalances(address, rewardProgramId);
-    console.log('\n');
-    console.log(`Reward balances for ${address}`);
-    console.log('---------------------------------------------------------------------');
-    displayRewardTokenBalance(tokenBalances);
-  } else {
-    const tokenBalances = await rewardPool.rewardTokenBalances(address);
-    console.log('\n');
-    console.log(`Reward balances for ${address}`);
-    console.log('---------------------------------------------------------------------');
-    displayRewardTokenBalance(tokenBalances);
-  }
+  const tokenBalances = await rewardPool.rewardTokenBalances(address, rewardProgramId);
+  console.log('\n');
+  console.log(`Reward balances for ${address}`);
+  displayRewardTokenBalance(tokenBalances);
 }
 
 function displayRewardTokenBalance(tokenBalances: RewardTokenBalance[]): void {
   const groupedByRewardProgram = groupBy(tokenBalances, (a) => a.rewardProgramId);
   Object.keys(groupedByRewardProgram).map((rewardProgramId: string) => {
+    console.log('---------------------------------------------------------------------');
     console.log(`    RewardProgram: ${rewardProgramId}`);
     console.log('---------------------------------------------------------------------');
```

Comment:
if you use a ` then you can wrap new lines without having to repeat the console.log statements like this:
```js
console.log(`
---------------------------------------------------------------------
RewardProgram: ${rewardProgramId}
---------------------------------------------------------------------`);